mitiq/benchmarks/tests/test_mirror_circuits.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# Copyright (C) 2021 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from mitiq.benchmarks import mirror_circuits
+import networkx as nx
+import cirq
+import numpy as np
+import pytest
+from cirq.experiments.qubit_characterizations import _single_qubit_cliffords
+
+paulis = [cirq.X, cirq.Y, cirq.Z, cirq.I]
+single_q_cliffords = _single_qubit_cliffords().c1_in_xy
+single_cliffords = []
+for gates in single_q_cliffords:
+    for gate in gates:
+        single_cliffords.append(gate)
+cliffords = single_cliffords
+cliffords.append(cirq.CNOT)
+all_gates = paulis + cliffords
+all_gates.append(cirq.measure)
+
+@pytest.mark.parametrize("n", (0, 5))
+def test_random_paulis(n):
+    circuit = mirror_circuits.random_paulis(nqubits=n)
+    assert isinstance(circuit, cirq.Circuit)
+    assert len(circuit.all_qubits()) == n
+    assert len(list(circuit.all_operations())) == n
+    assert set(op.gate for op in circuit.all_operations()).issubset(paulis)
+    
+def test_edge_grab():
```

Comment:
This looks good. Using `pytest.mark.parameterize` as above would help with readability.

Are there significant differences between the three cases? Maybe you only need two, or maybe you don't need to paramterize at all.